Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -255,12 +257,17 @@ extension GestureManager: GestureHandlerDelegate {
         return mapboxMap.cameraState.zoom
     }
 
-    internal func pinchScaleChanged(with newScale: CGFloat, andAnchor anchor: CGPoint) {
-        mapboxMap.setCamera(to: CameraOptions(anchor: anchor, zoom: newScale))
+    internal func pinchChanged(with zoom: CGFloat, anchor: CGPoint, previousAnchor: CGPoint) {
```

Comment:
```suggestion
    internal func pinchChanged(withZoom zoom: CGFloat, anchor: CGPoint, previousAnchor: CGPoint) {
```

In this case, CGFloat doesn't provide sufficient context, so you're supposed to include the name…

<img width="749" alt="Screen Shot 2021-07-22 at 2 47 49 PM" src="https://user-images.githubusercontent.com/205658/126700595-7fda060a-96c2-42cb-a215-b06dbde9e88b.png">

Reply:
also applies to pinchEnded below

Reply:
I will make this change but I want to point out that this is pretty inconsistent across the codebase. I've made this comment before but it would be great to have clarity on why/when we want to use argument labels. For example, this change requested above results in this:

`func pinchChanged(withZoom targetZoom: CGFloat, ...)`

but that feels redundant to me and I would typically prefer this:

`func pinchChanged(targetZoom: CGFloat, ...)`

In this case, does the argument label provide any value?  I think the primary reason people use argument labels is to avoid ambiguity but in this case, I feel that the argument label isn't necessary for that. Again, I will make this change but it feels like maybe the team needs to DDC on these kind of conventions.